src/js/components/bulkDownload/accounts/AccountDataContent.jsx

```diff
@@ -55,9 +55,7 @@ export default class AccountDataContent extends React.Component {
         this.props.clearAccountFilters();
     }
 
-    handleSubmit(e) {
-        e.preventDefault();
-
+    handleSubmit() {
```

Comment:
Looks like we could get rid of this function and just pass `this.props.clickedDownload` as to `SubmitButton`.